src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -553,18 +553,28 @@ public function text(string $default = null, bool $normalizeWhitespace = true):
         $text = $this->getNode(0)->nodeValue;
 
         if ($normalizeWhitespace) {
-            return trim(preg_replace("/(?:[ \n\r\t\x0C]{2,}+|[\n\r\t\x0C])/", ' ', $text), " \n\r\t\x0C");
+            return $this->normalizeWhitespace($text);
         }
 
         return $text;
     }
 
     /**
      * Returns only the inner text that is the direct descendent of the current node, excluding any child nodes.
+     *
+     * @param bool $normalizeWhitespace parameter not implemented yet
```

Comment:
`@param bool $normalizeWhitespace Whether whitespaces should be trimmed and normalized to single spaces`

Reply:
👍🏻 changed it. Still the fabbot.io github action doesn't like this phpdoc at all: https://fabbot.io/report/symfony/symfony/48940/83918eb8752ba86a76c34d4ecae52e525aa97e92